.github/workflows/pecl.yml

```
@@ -0,0 +1,80 @@
+name: Pecl builder
+
+on:
+  create:
+    tags:
+  push:
+    paths:
+      - '.github/workflows/pecl.yml'
```

```suggestion
      - .github/workflows/pecl.yml
```

Use quotes `'` only to _force_ a string.